apps/win32/metro.config.js

```diff
@@ -6,30 +6,47 @@
  */
 const path = require('path');
 const blacklist = require('metro-config/src/defaults/blacklist');
-const { getWatchFolders } = require('@uifabricshared/build-native');
+const {getWatchFolders} = require('@uifabricshared/build-native');
+const {getDefaultConfig} = require('metro-config');
 
-module.exports = {
-  watchFolders: getWatchFolders(),
-  resolver: {
-    resolveRequest: require('@office-iss/react-native-win32/metro-react-native-platform').reactNativePlatformResolver({
-      win32: '@office-iss/react-native-win32'
-    }),
-    blacklistRE: blacklist([
-      // This stops "react-native run-windows" from causing the metro server to crash if its already running
-      new RegExp(`${path.resolve(__dirname, 'windows').replace(/[/\\]/g, '/')}.*`),
-      // This prevents "react-native run-windows" from hitting: EBUSY: resource busy or locked, open msbuild.ProjectImports.zip
-      new RegExp(`${path.resolve(__dirname, 'msbuild.ProjectImports.zip').replace(/[/\\]/g, '/')}.*`)
-    ])
-  },
-  transformer: {
-    // The cli defaults this to a full path to react-native, which bypasses the reactNativePlatformResolver above
-    // Hopefully we can fix the default in the future
-    assetRegistryPath: 'react-native/Libraries/Image/AssetRegistry',
-    getTransformOptions: async () => ({
-      transform: {
-        experimentalImportSupport: false,
-        inlineRequires: false
-      }
-    })
-  }
-};
+
+module.exports = (async () => {
+    const {
+        resolver: {
+            sourceExts,
+            assetExts
+        }
+    } = await getDefaultConfig();
+    return {
+        watchFolders: getWatchFolders(),
+        resolver: {
+          assetExts: assetExts.filter(ext => ext !== 'svg'),
+          sourceExts: [
+              ...sourceExts,
+              'svg'
+          ],
+          resolveRequest: require('@office-iss/react-native-win32/metro-react-native-platform').reactNativePlatformResolver(
+              {win32: '@office-iss/react-native-win32'}
+          ),
+          blacklistRE: blacklist([
+              // This stops "react-native run-windows" from causing the metro server to crash if its already running
+              new RegExp(`${path.resolve(__dirname, 'windows').replace(/[/\\]/g, '/')}.*`),
+              // This prevents "react-native run-windows" from hitting: EBUSY: resource busy or locked, open msbuild.ProjectImports.zip
```

Comment:
I know that this was here before.  But this error is specific to the UWP version of RNW.  It shouldn't hit the win32 one., so this can be removed.

Reply:
Ok line removed.